src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -202,7 +202,9 @@ export const TradeConfirm = ({ history }: RouterProps) => {
     .times(bnOrZero(sellAssetFiatRate))
     .times(selectedCurrencyToUsdRate)
 
-  const protocolFeeCrypto = bnOrZero(fees?.tradeFee).div(bnOrZero(buyAssetFiatRate)).toString()
+  const protocolFeeCrypto = bnOrZero(fees?.sellAssetTradeFeeUsd)
```

Comment:
Possibly wrong
```suggestion
  const protocolFeeCrypto = bnOrZero(fees?.buyAssetTradeFeeUsd)
```

Reply:
Actually correct-ish, should account for agnostic side fees e.g subtract from the buy asset both in case the fees are taken from the buy / sell asset